orb/core/model.py

```diff
@@ -1159,6 +1159,12 @@ def fetch(cls, key, **context):
         else:
             context.setdefault('where', orb.Query(cls) == key)
 
+        # don't have slicing for lookup by id
```

Comment:
this was causing issues when fetching a record with paging, particularly when part of a traversal:

`/objects/1/sub-objects?page2`